publicatiewijzer-voor-de-archeologie.csl

```diff
@@ -64,8 +69,8 @@
     </names>
   </macro>
   <macro name="author-short">
-    <names variable="author">
-      <name form="short" and="text">
+    <names variable="author" delimiter="/">
```

Comment:
I'll fix this here, but for future reference: the delimiter on `names` is the delimiter between different author types (e.g. if you have `<name variable="author editor">`). It doesn't do anything if only a single creator type is given, so no purpose for it here. The one you want is in `name`

Reply:
Thanks for correcting and explaining! And I'm sorry for the confusion